db/static_seeds.rb

```diff
@@ -8,3 +8,6 @@
   {start_date: "2017-05-28", name: "Bio", video: nil, duration: nil},
   {start_date: "2017-05-28", name: "Konventionell", video: nil, duration: nil}
 ])
+Sensor.create!([
```

Comment:
Yes, but the sensors need to be created somewhere and I thought the static_seeds file was the right place for that. If each sensor gets an animal_id (see above) I need to create the specific sensor with all the parameters at some point (preferably when setting up the whole system, before the querying of the API and creating of sensor readings starts). Where else can I create the sensors?